content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -77,11 +77,11 @@ cosign verify-blob "$BINARY" \
 <!-- 
 Cosign 2.0 requires the `--certificate-identity` and `--certificate-oidc-issuer` options.
 
-To learn more about keyless signing, please refer to [Keyless Signatures](https://docs.sigstore.dev/cosign/keyless).
+To learn more about keyless signing, please refer to [Keyless Signatures](https://docs.sigstore.dev/signing/overview/).
```

Comment:
It appears that updating the `Keyless Signatures` link on [Line 89](https://github.com/kubernetes/website/blob/3ca006943fb895d0876bbd5cd4017f4f66c8ff87/content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md?plain=1#L89) might also be needed.

Reply:
Yes, that is required.

Reply:
Done